modular_skyrat/modules/gunsgalore/code/guns/fg42.dm

```diff
@@ -31,9 +31,6 @@
 
 	AddComponent(/datum/component/automatic_fire, fire_delay)
 
-/obj/item/gun/ballistic/automatic/fg42/give_manufacturer_examine()
-	AddComponent(/datum/component/manufacturer_examine, COMPANY_OLDARMS)
```

Comment:
I noticed that a lot of OLDARMS cases were removed rather than elementized. Intentional?

Reply:
Most of these guns haven't been produced new since at least like, 610 years, no modern company in 2563 is still making these or has made them for long enough for the markings to be discernable.

Reply:
Alrighty. Is that all the uses of company_oldarms removed, then? Could probably just remove it entirely

Reply:
I'm not sure actually, let me check